packages/cdktf-cli/src/test/ui/synth.test.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import React from "react";
+import { render } from "ink-testing-library";
+import { stripAnsi } from "../test-helper";
+import { SynthOutput } from "../../bin/cmds/ui/synth";
+import { SynthesizedStack } from "@cdktf/cli-core";
+
+test("SynthOutput", () => {
+  const { lastFrame } = render(<SynthOutput stacks={[]} />);
+  expect(stripAnsi(lastFrame())).toBe("No stacks found in configuration.");
+  {
```

Comment:
I'd rather this be split into two tests. While it is completely fine, I'm personally not a big fan of restricting scope like this. That being said, if you want there to be just one test, you can use `let` instead of `const`. We do that a lot where we want to test a variety of small changes within the same test.